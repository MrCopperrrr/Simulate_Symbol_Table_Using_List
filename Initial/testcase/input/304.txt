BEGIN
INSERT r string
PRINT
BEGIN
INSERT s number
PRINT
INSERT t string
END
PRINT
END
PRINT