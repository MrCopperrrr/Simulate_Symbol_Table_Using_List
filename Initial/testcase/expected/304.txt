success
r//1
success
r//1 s//2
success
r//1
